core/assets/bundles/bundle_nl.properties

```diff
@@ -118,619 +134,654 @@ players = {0} Spelers online
+zone.stainedMountains.description = Verder inland liggen de bergen, nog niet berijkt door schimmels.\nVerzamel de grote hoeveelheiden titanium in het gebied. Leer hoe je het kan gebruiken.\n\nDe vijand is krachtig hier. Geef ze geen tijd om je te overrompelen.
+zone.overgrowth.description = Dit gebied is overgroeid, dichter bij de bron van de schimmels.\nDe vijand heeft hier een uitkijkpost. Bouw dolk units. Vernietig hen. Claim was verloren was.
+zone.tarFields.description = De randen van een olie productie veld, tussen de bergen en de woestijn. Een van de weinige plekken met bruikbare olie.\nOndanks dat het verbannen is, zijn er wel krachtige vijanden in de buurt. Onderschat ze niet.\n\n[lightgray]Onderzoek wat je met olie kan doen indien mogelijk.
+zone.desolateRift.description = Een super gevaarlijk gebied. Veel grondstoffen, maar weinig ruimte. Grote kans op verwoesting. Lanceer zo snel mogelijk. Word niet overmoedig door de lange tijd tussen de rondes.
+zone.nuclearComplex.description = Een voormalige instalatie voor de productie en verwerking van thorium, ligt er nu verlaten bij.\n[lightgray]Onderzoek thorium en de vele toepassingen ervoor.\n\nDe vijand is aanwezig hier in grote getalen, constant aan het zoeken voor aanvallers.
+zone.fungalPass.description = Een transitiegebied tussen de hoger bergen en de lagergelegen, beschimmelde gebieden. Een kleine verkenningsbasis is hier gestationeerd.\nVernietig het.\nGebruik Dolk en Kruiper units. Maak de twee cores onbruikbaar.
 zone.impact0078.description = <insert description here>
 zone.crags.description = <insert description here>
-settings.language = Language
+settings.language = Taal
 settings.data = Game Data
-settings.reset = Reset to Defaults
-settings.rebind = Rebind
-settings.controls = Controls
-settings.game = Game
-settings.sound = Sound
-settings.graphics = Graphics
-settings.cleardata = Clear Game Data...
-settings.clear.confirm = Are you sure you want to clear this data?\nWhat is done cannot be undone!
-settings.clearall.confirm = [scarlet]WARNING![]\nThis will clear all data, including saves, maps, unlocks and keybinds.\nOnce you press 'ok' the game will wipe all data and automatically exit.
-paused = [accent]< Paused >
-clear = Clear
-banned = [scarlet]Banned
-yes = Yes
-no = No
-info.title = Info
-error.title = [crimson]An error has occured
-error.crashtitle = An error has occured
+settings.reset = Terug naar fabrieksinstellingen
+settings.rebind = Verander
+settings.resetKey = Reset
+settings.controls = Bediening
+settings.game = Spel
+settings.sound = Geluid
+settings.graphics = Grafisch
+settings.cleardata = Wis Game Data...
+settings.clear.confirm = Weet je zeker dat je deze data wilt verwijderen?\nDit is niet terug te draaien!
+settings.clearall.confirm = [scarlet]WAARSCHUWING![]\nDit verwijderd alle darta, inclusief saves, mappen, technologie en toetsen.\nAls je op doorgaat wist de game al je data en stopt ie automatisch.
```

Comment:
`data`
`toetsen` makes me think of school. (Suggestion: controle-instellingen)

The last part of this translation needs a rewrite.